content/docs/how-we-work/baseline/git-workflow.md

```diff
@@ -30,3 +30,25 @@ The branches can be deleted immediately and [automatically](https://docs.github.
 [GitHub provides a variety of protection mechanism](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches) for different branches. It especially recommends that the main branch (often called `main` or `master`) is protected from accidental pushes.
 
 We recommend that the default branch is always protected. At a minimum, a PR should be required before merging to the main branch. You can also enforce conditions such as the PR must have approvals and requiring reviews from code owners. More rules can be found in the [documentation](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches).
+
+### Number of branches
+
+It is recommended to have minimal number of branches as highlighted above. However below can be some scenarios based on which you can decide what works for your project.
+
+#### Scenario 1 (1 branch)
+
+This is the most ideal scenario where we have only 1 branch which is either `main` or `master`. Any other branches are temporary feature branches which would be deleted after the PR is closed. In case of any feature that needs to be reverted, raising a revert PR against the merged one would work.The `main` branch is linked to the Development environment where the next code for Production is pushed and QA is done followed by bug fixes.
```

Comment:
```suggestion
This is the ideal scenario where we have only one branch (usually `main`). Any other branches are temporary feature branches which would be deleted after the PR is closed. In case of any feature that needs to be reverted, raising a revert PR against the merged one would work. The `main` branch is linked to the Development environment where the next code for Production is pushed and QA is done followed by bug fixes. Individual feature environments are created per branch as required.
```

Reply:
Fixed in 8736b9aa7de8b9982aa0870972d77b6f6afe76b4 and removed above line as well